src/features/lightspeed/roleGeneration.ts

```diff
@@ -238,11 +238,12 @@ export async function showRoleGenerationPage(extensionUri: vscode.Uri) {
       }
       case "openEditor": {
         const { playbook } = message;
-        await openNewPlaybookEditor(playbook);
         await sendActionEvent(
           PlaybookGenerationActionType.CLOSE_ACCEPT,
           undefined,
         );
+        await openNewPlaybookEditor(playbook);
+        await openNewPlaybookEditor(playbook);
```

Comment:
Are these lines intentionally duplicated?

Reply:
I got an explanation from @goneri and understand that it was intentionally added. Resolving the conversation.